package/yast2.spec

```diff
@@ -137,6 +137,7 @@ This package contains scripts for handling YAST logs.
 %setup -q
 
 %check
+export Y2STRICTTEXTDOMAIN=1
```

Comment:
We should check all other packages, fix the possible issues and make this the default...

Reply:
Yes, that makes sense. We need to make a staging (local?) build of all packages first.